cmd/mqtt2prometheus.go

```diff
@@ -91,6 +91,7 @@ func main() {
 		mqttClientOptions.SetTLSConfig(tlsconfig)
 	}
 
+	fmt.Printf("[CONFIG] Timeout %v\n", cfg.Cache.Timeout)
```

Comment:
Can you either remove this print or replace it by a zap log entry on debug level?

Reply:
Oh, good catch. That was a poor-man debugging line I accidently commited. 
Removed.